packages/cli/src/index.ts

```diff
@@ -115,10 +115,13 @@ async function execCmd(args: string[]) {
         {
           yarn: "yarn",
           pnpm: "pnpm",
+          lerna: "lerna",
           root: "yarn",
           bolt: "bolt"
         }[tool],
-        tool === "pnpm" ? ["install"] : [],
+          tool === "pnpm" ? ["install"]
+        : tool === "lerna" ? ["bootstrap", "--since", "HEAD"]
```

Comment:
What does `--since HEAD` do?

Reply:
It ensures that bootstrap will only run against the packages changed by the fixes (and their local direct dependents) instead of the entire monorepo, in most cases.